spec/ttml1.xml

```diff
@@ -7107,6 +7107,9 @@ context of a layout or <loc href="#element-vocab-type-content">Content</loc> ele
 context, and not in the non-presentation, declaration context
 of a referentiable <el>style</el> element.</p>
 </note>
+<note role="clarification">
```

Comment:
This addition is also problematic and not adequately motivated or contextualized. Need to reconsider as suggested above.

Reply:
The idea is to remind the reader that the value "normal" is inherited, instead of its computed value, since it is not a relative value type.

Reply:
I don't see any text specific to line height in the CSS inheritance rule: https://www.w3.org/TR/css3-cascade/#inheriting

Reply:
See last paragraph of Section 5.1.2 at https://www.w3.org/TR/xsl11/